velox/functions/FunctionRegistry.h

```diff
@@ -95,4 +97,20 @@ resolveVectorFunctionWithMetadata(
 /// Clears the function registry.
 void clearFunctionRegistry();
 
+/// A function name is a companion function's if the name is an existing
+/// aggregation function name followed by one of specific suffixes.
+bool isCompanionFunctionName(
+    const std::string& name,
+    const std::unordered_map<std::string, exec::AggregateFunctionEntry>&
+        aggregateFunctions);
+
+/// The following helper functions return the list of scalar, aggregate, and
+/// window function names registered in Velox. The list can be sorted by setting
+/// the parameter sortNames to true.
+std::vector<std::string> getScalarNames(const bool sortNames = false);
```

Comment:
Would be great to add some tests for these functions.

Reply:
Thanks @aditi-pandit, added tests for these functions. Could you please take another look?